extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -549,29 +531,17 @@ Glyph createGlyph (char c, FreeTypeBitmapFontData data, FreeTypeFontParameter pa
 
 		}
 
-		String name = packPrefix + c;
-		Rectangle rect = packer.pack(name, mainPixmap);
+		Rectangle rect = packer.packDirectToTexture(mainPixmap);
 
 		// determine which page it was packed into
-		int pageIndex = packer.getPageIndex(name);
-		if (pageIndex == -1) // we should not get here
-			throw new IllegalStateException("Packer was not able to insert glyph into a page: " + name);
-
-		glyph.page = pageIndex;
+		glyph.page = packer.getPages().size - 1;

```

Comment:
I'm not sure if the packing algorithm has changed (I didn't change it), but by my reading in the current version it will always be the last page.  A new Pixmap passed to `pack` will only check `currPage` for space, and if it doesn't fit a new page is created, `currPage` is updated to point to the new page, and it tries to pack again.  I agree that it would result in a more efficient pack if `pack` scanned all pages for space for its Pixmap, but I don't believe that's how it currently works.